tools/chapel-py/src/core-types.cpp

```diff
@@ -395,6 +382,7 @@ static PyMethodDef AstNodeObject_methods[] = {
   {"pragmas", (PyCFunction) AstNodeObject_pragmas, METH_NOARGS, "Get the pragmas of this AST node"},
   {"unique_id", (PyCFunction) AstNodeObject_unique_id, METH_NOARGS, "Get a unique identifier for this AST node"},
   {"scope", (PyCFunction) AstNodeObject_scope, METH_NOARGS, "Get the scope for this AST node"},
+  {"type", (PyCFunction) AstNodeObject_type, METH_NOARGS, "Get the type of this AST node"},
```

Comment:
This also needs to be added to `generate-pyi.py`. Especially because for now it returns an unnamed tuple. Currently outside of the PR message, there is no description of what this returns. In the long run, this should definitely be a simple wrapper class or even a namedtuple, but for now its fine.